.github/workflows/main.yaml

```diff
@@ -0,0 +1,72 @@
+---
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  pull_request:
+    branches: [master]
```

Comment:
@bznein can we rename our branch to main? 🙏

Reply:
I think we need to wait for github to change the terminology to main? 

Or I think there's a way to change ourself the name of the branch to main, but in any case that should happen before we can change it here, otherwise it will never trigger as it won't see any branch named `main`

Reply:
@LouisPlisso it seems rather straightforward to rename it to main: https://dev.to/rhymu8354/git-renaming-the-master-branch-137b